src/niswitch/metadata/functions_addon.py

```diff
@@ -85,11 +85,11 @@
 
 functions_converters = {
     'ConfigureScanTrigger':              { 'parameters': { 1: { 'python_api_converter_name': 'convert_timedelta_to_seconds',
-                                                                'python_api_converter_type': 'datetime.timedelta', }, }, },
+                                                                'python_api_converter_type': 'float in seconds or datetime.timedelta', }, }, },
```

Comment:
I think it's misleading to call this piece of metadata `'python_api_converter_type'` when it does not contain a valid type. All it contains is a snippet used by documentation.

Since we have `'python_api_converter_name'` which is the name of the converter function used, this could be called something like `'python_api_converter_type_doc'` or something similar.

Reply:
I am calling it 'type_in_documentation'. Those are left over and didn't get changed to 'python_type' when the other did. There was some inconsistency between drivers that this will fix.